ITEM_en.tsv

```diff
@@ -3531,22 +3531,22 @@ ITEM_20150317_003736	Complete scriptures
 ITEM_20150317_003737	Scriptures contained the holy word
 ITEM_20150317_003738	Scriptures of soul
 ITEM_20150317_003739	Scriptures contained the Bishop's soul. It is able to recall the bishop to use it anytime.
-ITEM_20150317_003740	Relice of Mercy
+ITEM_20150317_003740	Relic of Mercy
 ITEM_20150317_003741	Jewel with the form of water drop. It is required to find Mayburn's key.
-ITEM_20150317_003742	Relice of rescue
+ITEM_20150317_003742	Relic of rescue
 ITEM_20150317_003743	Jewel with round figure. It is required to find Mayburn's key.
 ITEM_20150317_003744	The first key of Mayburn
 ITEM_20150317_003745	Relics of guards
 ITEM_20150317_003747	Bag contained the farming tool
 ITEM_20150317_003748	Bag fully contained with farming tools
 ITEM_20150317_003749	Bee Tree Branch
-ITEM_20150317_003750	Branches of bee tree contains the power of Goddness Austeza
+ITEM_20150317_003750	Branches of bee tree contains the power of Goddess Austeza
 ITEM_20150317_003751	Ashes of burned demon
-ITEM_20150317_003752	Ashes of devil burned due to Goddness Austeza's power
+ITEM_20150317_003752	Ashes of devil burned due to Goddess Austeza's power
 ITEM_20150317_003753	Beads of residue
 ITEM_20150317_003756	Materials to make scarecrow
 ITEM_20150317_003761	The third revelation found from Thorn Forest
-ITEM_20150317_003762	The fourth revelation found from Royaltobm
+ITEM_20150317_003762	The fourth revelation found from Royaltomb
```

Comment:
I just fixed typo. There is 32 "Royal Tomb" in files and  12 "RoyalTomb" didn't check if Korean version is the same.